### ISSUE TITLE
Included ansible version is a little old?

### ISSUE THREAD

**Issue by infraweavers** [ConSol/omd#72]
Hi Guys,

We're looking at PR'ing in ansible 2.7 rather than the 2.3 branch that's current in OMD; before we start on the actual work to do that, is there a specific reason that omd still ships out with 2.3? Alternatively is there any objection to an upgrade?

Many Thanks,
Rob

**Comment by sni** [comment]
We are planning an update for the next OMD 3.0 release. Since ansible updates are not backwards compatible, we postponed an update to the next major release.

**Comment by infraweavers** [comment]
Awesome, we'll wait for that then. Roughly is that due around summer 2019?

**Comment by sni** [comment]
I'd say around march 2019.
